staramr/detection/AMRDetection.py

```diff
@@ -97,8 +101,38 @@ def run_amr_detection(self, files, pid_threshold, plength_threshold_resfinder, p
         :param plength_threshold_pointfinder: The percent length overlap for BLAST results (pointfinder).
         :param plength_threshold_plasmidfinder: The percent length overlap for BLAST results (plasmidfinder).
         :param report_all: Whether or not to report all blast hits.
+        :param ignore_invalid_files: Skips the invalid input files if set.
         :return: None
         """
+
+        total_files = len(files)
+        invalid_files = 0
+        removeable_files = []
+
+        for file in files:
+            try:
+                # Tries to return a single valid sequence, will raise an error if the input is incorrect or has more than one
+                SeqIO.read(file, "fasta")
```

Comment:
I think you will want to use `SeqIO.parse()` instead of `SeqIO.read()` here. `SeqIO.read()` is only used when the FASTA file contains one and only one record (and the input files we expect will likely contain multiple records).

Reply:
I just noticed that now, with parse, i no longer need the try catch statement will resolve this in my next commit